toolchains/cc/cc.bzl

```diff
@@ -122,8 +122,8 @@ def _parse_cc_toolchain_info(content, filename):
 
 def _nixpkgs_cc_toolchain_config_impl(repository_ctx):
     host_cpu = get_cpu_value(repository_ctx)
-    cross_cpu = repository_ctx.attr.cross_cpu or host_cpu
-    darwin = (host_cpu == "darwin" or host_cpu == "darwin_arm64") and cross_cpu == host_cpu
+    cpu_value = repository_ctx.attr.cross_cpu or host_cpu
+    darwin = cpu_value == "darwin" or cpu_value == "darwin_arm64"
```

Comment:
Yes, I think this is correct and will ensure the correct version of _cc_wrapper is selected.